test/transaction.ts

```diff
@@ -32,8 +41,8 @@ const fakePfy = Object.assign({}, pfy, {
   },
 });
 
-// tslint:disable-next-line no-any variable-name
-const DatastoreRequestOverride: any = {
+// tslint:disable-next-line variable-name
+const DatastoreRequestOverride: Any = {
```

Comment:
Instead of `any` here, can we do ` as {} as DatastoreRequest` or something to that effect?